src/OrchardCore.Modules/OrchardCore.OpenId/Permissions.cs

```diff
@@ -4,30 +4,35 @@ namespace OrchardCore.OpenId;
 
 public sealed class Permissions : IPermissionProvider
 {
+    private readonly IEnumerable<Permission> _allPermissions =
+    [
+        OpenIdPermissions.ManageApplications,
+        OpenIdPermissions.ManageScopes,
+        OpenIdPermissions.ManageClientSettings,
+        OpenIdPermissions.ManageServerSettings,
+        OpenIdPermissions.ManageValidationSettings,
+    ];
+
+    [Obsolete("This will be removed in a future release. Instead use 'OpenIdPermissions.ManageApplications'.")]
     public static readonly Permission ManageApplications
```

Comment:
Please assign the old properties to the new static properties instead of creating a new instance.